src/custom-elements/docs/docs.json

```diff
@@ -1,5 +1,5 @@
 {
-  "timestamp": "2019-07-22T15:28:05",
+  "timestamp": "2019-08-20T10:36:44",
```

Comment:
@davidojedacl can you remove this file from your PR?

Reply:
i think is necessary after make tests